### PR TITLE
Update cloud-enabling-continuous-integration-with-github.md

### PR DESCRIPTION
**Pull request by jeremyyeo**
## Description & motivation
@mirnawong1 we missed out some stuff from the previous limitations PR #1871 

Also added some examples.

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by mirnawong1** [comment]
> ## Description & motivation
> @mirnawong1 we missed out some stuff from the previous limitations PR #1871
> 
> Also added some examples.
> 
> ## Checklist
> If you added new pages (delete if not applicable):
> 
> * [ ]  The page has been added to `website/sidebars.js`
> * [ ]  The new page has a unique filename
> 
> If you removed existing pages (delete if not applicable):
> 
> * [ ]  The page has been removed from `website/sidebars.js`
> * [ ]  An entry has been added to `_redirects`
> * [ ]  [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

gotcha thanks for catching that @jeremyyeo and will go ahead and approve + merge for you. thank yoU!

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md [hunk 1]
```diff
@@ -125,7 +125,8 @@ If your temporary PR schemas aren't dropping after a merge or close of the PR, y
 
 - You _have not_ used dbt Cloud environment variables in your connection settings page.
 - You _do not_ have an empty/blank default schema.
-- You _have not_ overridden the `generate_schema_name` macro.
+- You _have not_ overridden the `generate_schema_name` macro such that the temporary PR schema names no longer contain the prefix of `dbt_cloud_pr_`.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md]:
```suggestion
- You _have not_ overridden the `generate_schema_name` macro. 
If the temporary PR schema names don't contain the prefix of `dbt_cloud_pr_`, you most likely have overridden the macro and need to make changes. 
If they do contain the prefix, then you have not overridden the macro. 
```

**Comment by jeremyyeo** [comment]
Chatted with Mirna, I think we can use some better wording or examples? We can override the `generate_x_name` macros but if we override them into such scenarios, we can no longer drop the temp prs...

### Overriding `generate_schema_name` macro.

The default schema for a CI run takes on the form of `dbt_cloud_pr_123_456` where `123` is the dbt Cloud Job ID and `456` is the PR/MR number.

| Good                                             | Bad                                                                           |
| ------------------------------------------------ | ----------------------------------------------------------------------------- |
| Schema name is the default or contains the default as a prefix (e.g. `dbt_cloud_pr_123_456_marketing`. | Schema name does not have the default as a prefix (e.g. `marketing`). |

### Overriding `generate_database_name` macro.

Assuming that the project's default connection is to a database (or project if BigQuery) named `analytics`.

| Good                                                                    | Bad                                                 |
| ----------------------------------------------------------------------- | --------------------------------------------------- |
| Database remains the same as the connection default (i.e. `analytics`). | Database has changed from the default (e.g. `dev`). |

**Comment by mirnawong1** [comment]
i'm going to change the wording for sure because it's a bit confusing but I'll tag you for a review @jeremyyeo

**Review by mirnawong1 — APPROVED** [review]

**Pull request merged**